app/controllers/agenda_controller.rb

```diff
@@ -0,0 +1,15 @@
+class AgendaController < ApplicationController
+  helper_method :agenda
+
+  def show; end
+
+  private
+
+  def event
+    @event ||= Event.current.first!
```

Comment:
I think it's better to create `self` method instead of a scope and move `first!` there. (like with `upcoming` event)